### PR TITLE
Add/growth to comparison page

### PR DESCRIPTION
**Pull request by CodeyGuyDylan**
DON'T SHIP UNTIL DEC 3RD WHEN JETPACK GROWTH IS LAUNCHED

## Proposed Changes

* Add Jetpack Growth to comparison table

## Why are these changes being made?

* With a new bundle, we need to add Growth to the page comparing bundles

## Testing Instructions

1. Checkout this branch locally or use the Calypso Green live link
2. Go to `/features/comparison` in Jetpack Cloud
3. Make sure Growth is added and all the features are accurate
image

## Pre-merge Checklist

- [ ] Has the general commit checklist been followed? (PCYsg-hS-p2)
- [ ] [Have you written new tests](https://wpcalypso.wordpress.com/devdocs/docs/testing/index.md) for your changes?
- [ ] Have you tested the feature in Simple (P9HQHe-k8-p2), Atomic (P9HQHe-jW-p2), and self-hosted Jetpack sites (PCYsg-g6b-p2)?
- [ ] Have you checked for TypeScript, React or other console errors?
- [ ] Have you used memoizing on expensive computations? More info in [Memoizing with create-selector](https://github.com/Automattic/wp-calypso/blob/trunk/packages/state-utils/src/create-selector/README.md) and [Using memoizing selectors](https://react-redux.js.org/api/hooks#using-memoizing-selectors) and [Our Approach to Data](https://github.com/Automattic/wp-calypso/blob/trunk/docs/our-approach-to-data.md)
- [ ] Have we added the "[Status] String Freeze" label as soon as any new strings were ready for translation (p4TIVU-5Jq-p2)?
    - [ ] For UI changes, have we tested the change in various languages (for example, ES, PT, FR, or DE)? The length of text and words vary significantly between languages. 
- [ ] For changes affecting Jetpack: Have we added the "[Status] Needs Privacy Updates" label if this pull request changes what data or activity we track or use (p4TIVU-aUh-p2)?

### CODE REVIEW

File: client/jetpack-cloud/sections/comparison/table/style.scss [hunk 1]
```diff
@@ -322,7 +322,7 @@
 		width: 200px;
 	}
 
-	col.product-jetpack-starter {
+	col.product-jetpack-growth {
```
Comment (grzegorz-cp) [inline client/jetpack-cloud/sections/comparison/table/style.scss:324]: Did you mean to replace this or add another?

**Review by grzegorz-cp — APPROVED** [review]
👍

File: client/jetpack-cloud/sections/comparison/table/style.scss (hunk 1 above)
Comment (CodeyGuyDylan) [reply client/jetpack-cloud/sections/comparison/table/style.scss:324]: I meant to replace it. Starter is deprecated and isn't used on this page anymore, but I wanted to have the same styles as it, as it fit well with the others

**Comment by a8ci18n** [comment]
Translation for this Pull Request [has now been finished](https://translate.wordpress.com/deliverables/overview/17046055).

**Pull request merged**